Resources/doc/index.rst

```diff
@@ -0,0 +1,152 @@
+Router Bundle
+=============
+
+Contents:
+
+.. toctree::
+    :maxdepth: 150
+    :titlesonly:
+    :glob:
+
+    *
+    
+## Introduction
```

Comment:
Shouldn't this be a subsection? (http://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#headings)